### DISCUSSION TITLE
Message polled from server

### DISCUSSION THREAD

**Discussion by liuyinze** [khoih-prog/WebSockets2_Generic#48] [Q&A]
Hi all,

It's me again. This time I do have obtained something from server and everything seems to be "fine". But actually some important message is missing and I don't know why. 

Here is what I have achieved: 
1. Using const char* websockets_server_host = "wss://env779703.eu-latest.cumulocity.com/notification2/consumer/?token=mytoken&consumer=testSubscriber"  to connect.
2. Obtained the response from server as following: 

```
 {"self":"https://env779703.eu-latest.cumulocity.com/measurement/measurements/52344","time":"2022-07-27T15:46:44.000+02:00","id":"52344","source":{"self":"https://env779703.eu-latest.cumulocity.com/inventory/managedObjects/1825","id":"1825"},"type":"c8y_SignalStrengthWifi"}
COqyBBAlIAAwAQ==
/env779703/measurements/1825
CREATE

```  
And I tried with python with websocket-client package. The result is as following:

```
CJi0BBDYByAAMAE=
/env779703/measurements/1825
CREATE
{"self":"https://env779703.eu-latest.cumulocity.com/measurement/measurements/71828","time":"2022-07-28T10:34:36.000+02:00","id":"71828","source":{"self":"https://env779703.eu-latest.cumulocity.com/inventory/managedObjects/1825","id":"1825"},"type":"c8y_SignalStrengthWifi","c8y_SignalStrengthWifi":{"rssi":{"unit":"dBm","value":-63.0}}}
```
The part 
```
"c8y_SignalStrengthWifi":{"rssi":{"unit":"dBm","value":-63.0}}
```
is missing while using the library **WebSockets2_Generic**. 

I am wondering the body
```
{"self":"https://env779703.eu-latest.cumulocity.com/measurement/measurements/52344","time":"2022-07-27T15:46:44.000+02:00","id":"52344","source":{"self":"https://env779703.eu-latest.cumulocity.com/inventory/managedObjects/1825","id":"1825"},"type":"c8y_SignalStrengthWifi"}
```
is not the correct one or the correct one is not sent by the server?

It would be appreciated if someone can help me!

**Comment by khoih-prog** [comment]
Hi @liuyinze 

Sorry I don't have time or way to duplicate the issue. You're on your own now.

I suggest you post for help on Arduino Forum, with [**Minimal, Reproducible Example**](https://stackoverflow.com/help/minimal-reproducible-example). Otherwise, nobody can help without duplication.

If possible, you can also post the differences in **the requests sent to server** by this library and the `python` websocket-client.
